users/dshields/config.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define USB_MAX_POWER_CONSUMPTION 100
+#define ONESHOT_TAP_TOGGLE        2
+#define ONESHOT_TIMEOUT           3000
+#define RETRO_TAPPING
+
+#define MOUSEKEY_INTERVAL         20
+#define MOUSEKEY_DELAY            0
+#define MOUSEKEY_TIME_TO_MAX      40
+#define MOUSEKEY_MAX_SPEED        7
+#define MOUSEKEY_WHEEL_DELAY      0
+
+// dynamic macro keys
```

Comment:
All of the stuff below this line may be better suited for the `dshields.h` file, but that would require including the file in your keymap.c files.

Reply:
Thanks for the suggestion, I'll modify the commit.